### DISCUSSION TITLE
Setting collapsed state as a default

### DISCUSSION THREAD

**Discussion by developer9969** [H3ALY/CollapsibleNavMenu#1] [General]
Hi new to blazor wondering on you set the collapsed state as default when the app is launched below didnt work as I can still see text and the rest.. 

Many thanks!

  protected override void OnInitialized()
  {
      base.OnInitialized();
      ToggleIconMenu(true);
  }

**Comment by developer9969** [comment]
sorted - just as I posted lol

**Comment by H3ALY** [comment]
For others that wish to set the default state of the nav menu to collapsed update the following two files.

NavMenu.razor

Update `private bool IconMenuActive { get; set; } = false;` to `private bool IconMenuActive { get; set; } = true;`

MainLayout.razor

Update  `private bool IconMenuActive { get; set; } = false;` to `private bool IconMenuActive { get; set; } = true;`


I may look to consolidate this in the future.
